leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -1026,15 +1080,16 @@ private void closeKeyboard() {
     private String locationSearchZone;
     private String layoutsToHide = "";
     private boolean enableSatelliteView = true;
+    private boolean showLongitudeLatitude = false;
     private boolean shouldReturnOkOnBackPressed = false;
     private List<LekuPoi> lekuPois;
 
     public Builder() {
     }
 
-    public Builder withLocation(double latitude, double longitude) {
-      this.locationLatitude = latitude;
-      this.locationLongitude = longitude;
+    public Builder withLocation(double lat, double lng) {
```

Comment:
i was working with lat and long, and later I decided to rename all variables to be more uniform through out the lib